src/components/alert/alert.css

```diff
@@ -0,0 +1,29 @@
+.alert {
+  padding: 1.6rem;
+  max-width: 100rem;
+  display: flex;
+  align-items: center;
+  border-radius: 0.4rem;
+  margin: .8rem;
+  font-size: 1.6rem;
+}
+.alert-icon {
+  width: 3rem;
+  margin-right: .8rem;
+}
+.alert-primary {
+  background: #cfe2ff;
+  color: #084298;
```

Comment:
Using css variables here would have looked more consistent with the code

Reply:
thank you for the review, yes I will correct it.